extension-manifest-v3/src/background/onboarding.js

```diff
@@ -12,32 +12,45 @@
 import { observe } from '/store/options.js';
 import { sendShowIframeMessage } from './utils/iframe.js';
 import { tabStats } from './stats.js';
+import { showOperaSerpNotification } from '/utils/opera-serp.js';
```

Comment:
I think we can put in `notifications` folder (can be nested under some other folder but I would not call it `utils`, maybe `features`) - it is very likely we will have more of those. One that comes to mind would be a notification to enable Ghostery in incognito windows.

Reply:
ok, can be `notifications`.